spec/cli_spec.rb

```diff
@@ -1,12 +1,23 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'open3'
```

Comment:
Please alphabetize require statements.

```suggestion
require 'open3'
require 'spec_helper'
```